dbt_subprojects/tokens/macros/transfers/net_transfers.sql

```diff
@@ -0,0 +1,226 @@
+{% macro evm_net_transfers_daily(blockchain) %} 
+
+with raw_transfers as (
+    select
+        blockchain
+        , block_date
+        , "from" as address
+        , 'sent' as transfer_direction
+        , (sum(amount_usd) * -1) as transfer_amount_usd
+    from
+        {{ ref('tokens_transfers') }}
+    where
+        blockchain = '{{blockchain}}'
+        {% if is_incremental() or true %}
```

Comment:
```suggestion
        {% if is_incremental() %}
```

Reply:
only commenting this as a reminder to us to remove prior to finalizing, we can leave until then 🙏 
this will of course apply universally on the macro file